pkg/db/client.go

```diff
@@ -146,6 +169,57 @@ func (c *ClientSqlx) Exec(ctx context.Context, query string, args ...interface{}
 	return c.db.ExecContext(ctx, query, args...)
 }
 
+func (c *ClientSqlx) ExecMultiInTx(ctx context.Context, sqllers ...Sqller) (results []sql.Result, err error) {
```

Comment:
should we allow to pass an optional `*sql.TxOptions`?

Reply:
ya, I guess in 99% of the cases it is not needed so I would like to keep the signature simple for now